lib/puma/cluster.rb

```diff
@@ -386,7 +395,7 @@ def run
           end
         end
       else
-        log "*     Restarts: (\u2714) hot (\u2714) phased"
+        log "*     Restarts: (\u2714) hot (\u2714) phased (#{@options[:fork_worker] ? "\u2714" : "\u2716"}) refork"
```

Comment:
As of https://github.com/puma/puma/commit/e438b907def6702bd1ec35cbf8c623ba12c5e19e, refork is compatible with preloading, so here we just need to check if `fork_worker` has been enabled in the logs for both the preload case above and this non-preload case.